src/test/resources/testng.xml

```diff
@@ -237,6 +242,12 @@
       <class name="test.listeners.github1130.GitHub1130Test"/>
       <class name="test.listeners.github1296.GitHub1296Test"/>
       <class name="test.listeners.github1465.IssueTest"/>
+      <class name="test.listeners.github2385.TestInterfaceListenersInherit"/>
```

Comment:
It is not documented but we are no more adding TestNG test cases like that.
Instead, we are executing TestNG into TestNG.
Check https://github.com/cbeust/testng/blob/master/src/test/java/test/SimpleBaseTest.java